gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -463,22 +463,22 @@ const CssVariablesPage = () => {
             <ColorDocRow colorVar="--nx-color-global-sidebar-toggle">
               The color of the icon within the <NxCode>NxGlobalSidebar</NxCode> toggle button.
             </ColorDocRow>
-            <ColorDocRow colorVar="--nx-color-global-sidebar-toggle-background-hover">
+            <ColorDocRow colorVar="--nx-color-global-sidebar-background-hover">
```

Comment:
The description for the variable, it applies to more than just the toggle now correct?

Reply:
> The description for the variable, it applies to more than just the toggle now correct?

Ah yes, it's for both the toggle and nav links. Updated doc here: aecbb088625c87e17569544c80c8610f470702b2